packages/fx-core/src/plugins/resource/frontend/index.ts

```diff
@@ -100,6 +98,13 @@ export class FrontendPlugin implements Plugin, ArmResourcePlugin {
     );
   }
 
+  public async executeUserTask(func: Func, ctx: PluginContext): Promise<TeamsFxResult> {
+    FrontendPlugin.setContext(ctx);
+    return this.runWithErrorHandling(ctx, TelemetryEvent.MigrateV1Project, () =>
```

Comment:
NIT: Actually I think we would better send `execute-user-task` here and send another `migrate-v1-project` in the condition.

Reply:
Changed it to execute-user-task and add a property `method`